src/sidebar/views/sidebar.tsx

```diff
@@ -0,0 +1,55 @@
+import MenuItem from './menu-item';
+import React, { useState } from 'react';
+import SidebarItem from './menu-sub-item';
+import { AiFillLayout } from 'react-icons/ai';
+import { sidebarData } from 'sidebar/sidebar.data';
+import { FaChevronUp, FaChevronDown } from 'react-icons/fa';
+import { SidebarPropertyType, MenuKey, MenuItemType } from 'sidebar/sidebar.types';
+
+const { food, dashboard } = sidebarData;
+
+const menuItems = {
+  dashboard: {
+    name: 'DASHBOARD',
+    icon: <AiFillLayout />,
+    key: MenuKey.DASHBOARD,
+    child: <SidebarItem item={dashboard} />,
+  },
+  food: {
+    name: 'FOOD',
+    key: MenuKey.FOOD,
+    icon: <AiFillLayout />,
+    child: food.map((item: SidebarPropertyType, index: number) => <SidebarItem item={item} key={index} />),
+  },
+};
```

Comment:
These data eventually grows to larger number increasing the number lines in this file and make code dirty. Why not move them to `sidebar.menu-items.ts` where we add and remove these kinda data.